frontend/src/components/Profile/Task.js

```diff
@@ -0,0 +1,15 @@
+import classes from './Task.module.css';
+
+const Task = (props) => {
+  return (
+    <li className={classes.task}>
+      <h2>{props.user_id}</h2>
```

Comment:
i think this is too many `h2`s. let's make this a bit more semantically correct using different tags.